README.md

```diff
@@ -204,7 +229,7 @@ $.NonZeroValidNumber :: Type
 ```
 
 Type comprising every [`ValidNumber`](#validnumber) value except `0` and `-0`
-(and their object counterparts). 
+(and their object counterparts).
```

Comment:
There are a few more occurrences of trailing whitespace in this file. Mind trimming these while you're at it?